washingtontrails/vms
# Issues/72

This pull request fixes issue #72 by providing a pop-up modal for days with more than two events scheduled.  The modal is signaled by the text "see all *n* scheduled events" and when open, displays all events for the day using the same listing format shown on the "list" tab of the search page.

## To Test
1.  Ensure that some days have more than two events scheduled
2.  Click on the calendar tab of the search page
3.  Note that on a day with more than two events the text "See all *n* scheduled events" appears (where *n* is the number that are on that day)
4.  Click on the day (or on the text) to open the pop-up
5.  Verify that all events scheduled for that one day appear and that no other events do.
6.  Click any event to view the detailed listing for that event
7.  Click the 'close' (X) button at the top right of the pop-up to dismiss the pop-up and return to the calendar view
8.  Note that if you have moved to a non-default page of the calendar (some months in the future), that page is still displayed (you are viewing the same thing you were when you opened the pop-up)
